src/Microsoft.AspNetCore.Authorization.Policy/AuthorizationAppBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class AuthorizationAppBuilderExtensions
+    {
+        public static IApplicationBuilder UseAuthorization(this IApplicationBuilder app)
+        {
+            if (app == null)
+            {
+                throw new ArgumentNullException(nameof(app));
+            }
+
+            return app.UseMiddleware<AuthorizationMiddleware>();
+        }
+
+        public static IApplicationBuilder UseAuthorization(this IApplicationBuilder app, AuthorizationOptions options)
```

Comment:
Why do we need this overload?  I think its just got the potential for confusion when this overload will ignore the authorization options configured normally, is this needed for the static files sample?